python/python/lance/dataset.py

```diff
@@ -418,6 +425,8 @@ def to_table(
             Return row ID.
         use_stats: bool, default True
             Use stats pushdown during filters.
+        full_text_query: (str, str), optional
+            The string column to search and the query string to search for.
```

Comment:
Also, once we fully document as the above comment suggests let's add something like:

> full_text_query currently only supports a single column in the columns list